Topic4/src/interfaces/User.java

```diff
@@ -0,0 +1,24 @@
+package interfaces;
+
+import java.awt.Image;
+
+public interface User {
+
+	public void updateUser(String mail, String password, String name, String lastName);

```

Comment:
This method is not necessary in an interface, each class implementing this interface will have his own constructor and will be defined his own class variables, please remove